src/utils/star-canvas.ts

```diff
@@ -8,7 +8,7 @@ const isPhone = window.innerWidth < 450;
 
 const prefersReducedMotion = window.matchMedia(`(prefers-reduced-motion: reduce)`).matches;
 
-const scale = isMobile ? 1.5 : 3;
+const scale = isMobile ? 1.5 : 2.75;
```

Comment:
Had to make star slightly smaller on large screens to fit in the 90vh, but the makeup of the star looks of the same (it didn't fuck it up like previous rescales)